### ISSUE TITLE
Preprocess -> get Target file

### ISSUE THREAD

**Issue by lecorguille** [PF2-pasteur-fr/SARTools-Galaxy#6]
The Preprocess tool should for me just generate the target file and let SARTools deal with the tables.

Furthermore, Preprocess can be interesting for some other tools which need a target file (I'm thinking about a tool in the W4M workflow)

**Comment by hvaret** [comment]
Hi @lecorguille,
sorry for the late answer. If I remember well, the advantage of generating both the target and zip files was to avoid any error when running SARTools as we are sure that all the count files in the target will be in the zip archive.
Hugo

**Comment by lecorguille** [comment]
Hum ... maybe :/
The other possibility should be to merge the 3 tools: preprocess + SARTools_DESeq2 + SARTools_EdgeR

**Comment by hvaret** [comment]
Yes it could be nice to have only one tool (and may be easier to maintain) !
